lib/movable_ink/aws.rb

```diff
@@ -65,6 +65,8 @@ def run_with_backoff(quiet: false, tries: 9)
         rescue Aws::EC2::Errors::RequestLimitExceeded,
                Aws::EC2::Errors::ResourceAlreadyAssociated,
                Aws::EC2::Errors::Unavailable,
+               Aws::EC2::Errors::InternalError,
+               Aws::EC2::Errors::Http503Error,
```

Comment:
I'm a little wary of this change, do we do any mutations with the EC2 APIs in this gem or only reads?

Reply:
We just extend list of exception classes which are captured by begin/rescue here, e.g. to make classes which use `run_with_backoff` method (most of `miaws` ones use it when sending requests to AWS API) to retry if intermittent got from AWS. 

I've got the list of those classes from Slack messages sent in final `rescue` block:
![Screen Shot 2021-12-20 at 12 47 52 PM](https://user-images.githubusercontent.com/14876304/146810327-cd5b0f80-2590-4e1e-a6fd-3757e1af28f3.png)